scim2.yaml

```diff
@@ -1640,6 +1640,339 @@ paths:
+        403:
+          description: Forbidden
+          schema:
+            $ref: '#/definitions/ErrorForbidden'
+        404:
+          description: Valid role is not found
+          schema:
+            $ref: '#/definitions/ErrorNoRoleAvailable'
+    put:
+      tags:
+        - Roles Endpoint
+      summary: Update Role - PUT
+      description: |
+        This API updates the role details and returns the updated role details using a PUT operation.
+
+        <b>Permission required:</b>
+
+            * /permission/admin/manage/identity/rolemgt/update
+      operationId: updateRoleV3
+      consumes:
+        - application/scim+json
+      produces:
+        - application/scim+json
+      parameters:
+        - name: id
+          in: path
+          description: Unique ID of the resource type.
+          required: true
+          type: string
+        - in: body
+          name: body
+          required: false
+          schema:
+            $ref: '#/definitions/RolePutRequestObjectV3'
+      responses:
+        200:
+          description: Role is updated
+          schema:
+            $ref: '#/definitions/RolePutResponseObjectV3'
+        400:
+          description: Invalid Input
+          schema:
+            $ref: '#/definitions/ErrorInvalidInput'
+        401:
+          description: Unauthorized
+          schema:
+            $ref: '#/definitions/ErrorUnauthorized'
+        403:
+          description: Forbidden
+          schema:
+            $ref: '#/definitions/ErrorForbidden'
+        404:
+          description: Valid role is not found
+          schema:
+            $ref: '#/definitions/ErrorNoRoleAvailable'
+        406:
+          description: Not Acceptable
+          schema:
+            $ref: '#/definitions/ErrorNotAcceptable'
+    delete:
+      tags:
+        - Roles Endpoint
+      summary: Delete Role
+      description: |
+        This API deletes a particular role using its unique ID.
+
+        <b>Permission required:</b>
+
+            * /permission/admin/manage/identity/rolemgt/delete
+      operationId: deleteRoleV2
+      produces:
+        - application/scim+json
+      parameters:
+        - name: id
+          in: path
+          description: Unique ID of the resource type.
+          required: true
+          type: string
+      responses:
+        204:
+          description: Role is deleted
+        401:
+          description: Unauthorized
+          schema:
+            $ref: '#/definitions/ErrorUnauthorized'
+        403:
+          description: Forbidden
+          schema:
+            $ref: '#/definitions/ErrorForbidden'
+        404:
+          description: Valid role is not found
+          schema:
+            $ref: '#/definitions/ErrorNoRoleAvailable'
+    patch:
+      tags:
+        - Roles Endpoint
+      summary: Update Role - PATCH
+      description: |
+        This API updates the role details and returns the updated role details using a PATCH operation.
```

Comment:
Clearly mention the allowed attributes to be updated via this PATCH and PUT endpoints

Reply:
Fixed with: https://github.com/wso2-extensions/identity-inbound-provisioning-scim2/pull/645/commits/2db19bfd2bf84d38ea1c1116c7dd300d49b264d0